spec.md

```diff
@@ -489,7 +489,8 @@ When pushing a manifest with the `subject` field and the [referrers API](#listin
 1. If the tag returns a 404, the client MUST begin with an empty image index.
 1. Verify the descriptor for the manifest is not already in the referrers list (duplicate entries SHOULD NOT be created).
 1. Append a descriptor for the pushed manifest to the manifests in the referrers list.
-   The value of the `artifactType` MUST be set to the config descriptor `mediaType` in the image manifest.
+   The value of the `artifactType` MUST be set to the `artifactType` value in the image manifest.
```

Comment:
seems confusing that the artifactType MUST be set to the artifactType field of the image manifest .. or if nil/unset must be set to the config descriptor mediaType.. 

Indicates it may be set in both places and that they can be inconsistent..

If both we should talk about the rules here .. such that you can't have both and sometimes one is used and other times the other is used.. depending.  Does set/unset logic apply? I didn't check the schema.  I'm thinking if both are set they must be the same or it's an error.

Reply:
are we saying/proposing at least one of these must be set and not nil?

Reply:
I'm looking for the right wording to say artifactType must be set, and the value is the manifest artifactType if defined, otherwise it falls back to the config.mediaType, so there's a precedence to document. The config.mediaType is always defined in the image-spec.

Reply:
kk let's go to chat.. :-)